TwilioVerifySDK/TwilioVerify/Sources/Extensions/URLSession+Extensions.swift

```diff
@@ -48,8 +48,19 @@ extension URLSession {
   }
 }
 
-internal struct FailureResponse {
-  let responseCode: Int
-  let errorData: Data
-  let headers: [AnyHashable: Any]
+public struct APIError: Codable {
+  public let code: Int
+  public let message: String
+  public let moreInfo: String
```

Comment:
Do we know if any of these new properties could be optional?

Reply:
According to this: https://twilio-cms-prod.s3.amazonaws.com/documents/twilio-error-codes.json, more_info could be optional

Reply:
Thanks guys, just updated the model